continuous_integration/environment-3.10.yaml

```
@@ -0,0 +1,79 @@
+# This job includes coverage
+name: test-environment
+channels:
+  - conda-forge
+  - nodefaults
+dependencies:
+  # required dependencies
+  - python=3.10
+  - setuptools<60.0
+  - packaging
+  - numpy
+  - pandas
+  # test dependencies
+  - pre-commit
+  - pytest
+  - pytest-cov
+  - pytest-rerunfailures
+  - pytest-xdist
+  - moto
+  - flask
+  # Avoid `SystemError: PY_SSIZE_T_CLEAN macro must be defined for '#' formats`
+  # from thrift on Python 3.10
+  # https://github.com/dask/fastparquet/issues/514
+  # https://github.com/apache/thrift/pull/2487
+  # - fastparquet
```

AIUI the thrift issue was addressed in 0.16.0. Though fastparquet as of 0.8.0 no longer needs thrift. So should be able to add this back

```suggestion
  - fastparquet>=0.8.0
```